k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml

```diff
@@ -19,6 +19,7 @@ images:
     "sha256:a2049bd4253224693abb248ad446c633a2028e1af5b0628491af5ef7877a0166": ["v0.2.1"]
     "sha256:d5ca573e5fb723a52b267df51e00c811fe97f4d797fb8ad6f8680f13f3282008": ["v0.2.2"]
     "sha256:b6a3a5ee2821f91fbca65f1374997dd651470e0ca053bb61516c3354822f0ad9": ["v0.2.3"]
+    "sha256:c26073cd27cd53e51b96b8b73fb11fce0b077c196bd650c5e356fa721d8e1bc1": ["v0.2.4"]
 - name: cluster-api-controller-amd64
```

Comment:
Nit, but while this is being edited it might be nice to also remove the arch specific images below that aren't necessary for promoting the manifest list image and it's references.

Reply:
Sure thing!